FlowCrypt/src/main/java/com/flowcrypt/email/util/GeneralUtil.kt

```diff
@@ -429,7 +429,7 @@ class GeneralUtil {
     /**
      * Generate a FES url.
      */
-    fun generateFesUrl(domain: String): String {
+    fun generatePotentialCustomFesUrl(domain: String): String {
       return "https://fes.$domain/api/v1/client-configuration?domain=$domain"
     }
```

Comment:
Here this is unclear, because a FES URL should be something like `https://fes.$domain/` or `https://flowcrypt.com/shared-tenant-fes/`, depending on the situation (used to be https://fes.$domain/ vs https://flowcrypt.com/api with different APIs, but now the actual APIs are the same format).

So the proper name here would be `generatePotentialCustomFesUrlV1ClientConfigurationEndpointUrl`, but the functionality itself is not handy. You would have to do this for every FES endpoint twice, which makes no sense. You'd have to have:
 - `generatePotentialCustomFesClientConfigurationEndpointUrl`
 - `generateSharedTenantFesClientConfigurationEndpointUrl`
 - `generatePotentialCustomFesMessagePostUrl`
 - `generateSharedTenantFesMessagePostUrl`
 - ... and so on

What you should probably do instead is 

```kt
  fun generatePotentialCustomFesUrl(domain: String): String {
      return "https://fes.$domain"
    }
```

Then elsewhere have  `$actualFesUrl/api/v1/client-configuration?domain=$domain` where the actualFesUrl is either result of this `generatePotentialCustomFesUrl` or `SHARED_TENANT_FES_URL`, depending on the initial service discovery.